Directory.Build.props

```diff
@@ -307,7 +307,9 @@
 
   <!-- Language configuration -->
   <PropertyGroup>
-    <LangVersion>latest</LangVersion> <!-- default to allowing all language features -->
+    <!-- default to allowing all language features -->
+    <LangVersion>latest</LangVersion>
+    <LangVersion Condition="'$(Language)' == 'C#'">8.0</LangVersion>
```

Comment:
It's worth noting that should be able to set this to `preview` the next time around.

https://github.com/dotnet/roslyn/pull/33089